### PR TITLE
updating match attribute to mention XSLT 3 specs, broaden reference b…

### PR DESCRIPTION
**Pull request by ebeshero**
…eyond certainty re #2078

### CODE REVIEW

**Comment by ebeshero** [comment]
Seeking a review in time to merge this, to address #2078 in time for the Guidelines 4.2.0 release. Please make sure the references to the XSLT 3 specs are accurate and the tiny change to remarks is okay!

**Comment by ebeshero** [comment]
(I assigned four reviewers mainly just because we're getting close to refrigeration: probably if only one or two review it's okay. I just want someone to double-check the language and make sure it's accurate.)  @sydb @peterstadler @martinascholger  @hcayless

File: P5/Source/Guidelines/en/CE-CertaintyResponsibility.xml [hunk 1]
```diff
@@ -242,8 +242,8 @@ of the tagging throughout a document, rather than (as discussed so far) in one
 particular instance. The <att>match</att> attribute may be used to
 supply a pattern identifying the portion of a document
 concerning which certainty is being expressed. The value of the
-<att>match</att> attribute is an XPath expression <!--XSLT2 pattern,-->
-using the syntax defined in <ptr target="#XSLT2"/>. In the following
+<att>match</att> attribute is an XPath expression <!--XSLT3 pattern,-->
```
Comment (sydb) [inline P5/Source/Guidelines/en/CE-CertaintyResponsibility.xml]: I do not think this is correct. I think the value of `@match` is a selection pattern. See [the XSLT spec](https://www.w3.org/TR/xslt-30/#dt-pattern). Thus:
~~~xml
The value of the <att>match</att> attribute is an XSLT selection pattern using the syntax defined in <ptr target="#XSLT3"/>.
~~~

File: P5/Source/Specs/att.scoping.xml [hunk 2]
```diff
@@ -38,8 +38,8 @@ $Id$
         </egXML>
       </exemplum>
 
-      <remarks versionDate="2016-03-15" xml:lang="en">
-        <p>The expression of certainty applies to the nodeset identified by the value of the
+      <remarks versionDate="2021-01-30" xml:lang="en">
+        <p>The expression applies to the nodeset identified by the value of the
```
Comment (sydb) [inline P5/Source/Specs/att.scoping.xml]: I am uncomfortable with this. “The expression” seems ambiguous. I would prefer something like “The semantics of the element upon which this attribute appears apply to the …”.

**Review by sydb — COMMENTED** [review]
Nothing to do with the changes you have made so far, @ebeshero, but I also realize that the `<remarks>` in att.scoping are in the wrong place. They are remarks about both `@match` and `@target` (and thus, in some sense, about the entire class) but re inside the `<attDef ident="match">`; they should be a child of the `<classSpec>`, I think.

Furthermore, the description of `@target` leaves a bit to be desired:
> points at one or several elements or sets of elements by means of one or more data pointers, using the URI syntax.

I propose instead

> points at one or more sets of zero or more elements each.

Is it necessary to say anything about the syntax here?

File: P5/Source/Specs/att.scoping.xml (hunk 2 above)
Comment (ebeshero) [reply P5/Source/Specs/att.scoping.xml]: @sydb If I'm moving these remarks so they're a child of `classSpec`, it won't make sense to be discussing just the `@match` attribute, but rather the whole element. I'm trying just this (slightly simpler) phrasing: 
`The semantics of this element apply to the nodeset identified by the value of the
      <att>target</att> attribute, possibly modified by the value of the
      <att>match</att> attribute.`

Comment (ebeshero) [reply P5/Source/Specs/att.scoping.xml]: @sydb I totally agree about simplifying the `desc` for `@target`...implementing that.

**Comment by ebeshero** [comment]
@sydb I've updated my PR with some fresh edits thanks to your suggestions. Take another look?

**Comment by ebeshero** [comment]
Realized I needed to position the `<remarks>` to follow the `attDef`s. Fixed now.

**Comment by ebeshero** [comment]
@sydb @hcayless @peterstadler @martinascholger This PR should be ready to “press the button” and merge in...Just want to make sure it is not forgotten!

**Comment by sydb** [comment]
Review now …

**Comment by sydb** [comment]
(I had not forgotten, @ebeshero, just working on other PRs. :-)
Had to pull in changes from dev to get it to build, but seems to be working now.

**Pull request merged**